evap/staff/templates/staff_evaluation_textanswers_quick.html

```diff
@@ -29,114 +29,141 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
         </div>
     </div>
 
-    <div class="card card-outline-primary slider">
-        <div class="card-header">
-            {% trans 'Text answers' %}
-            <span class="hotkey-btn" aria-hidden="true" data-bs-toggle="tooltip" data-bs-placement="right" title="{% trans 'Hotkeys' %}">
-                <span class="far fa-keyboard" data-bs-toggle="modal" data-bs-target="#hotkeys-modal"></span>
-            </span>
-        </div>
-        <div class="card-body slider-inner px-0">
-            <div class="slider-side slider-side-left">
-                <span class="badge rounded-pill bg-primary" data-counter="reviewed-left" data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'Reviewed text answers' %}"></span>
-                <span class="fas fa-angle-left" data-slide="left"></span>
-                <span class="badge rounded-pill bg-secondary" data-counter="unreviewed-left" data-bs-toggle="tooltip" data-bs-placement="left" title="{% trans 'Unreviewed text answers' %}"></span>
+    <form method="POST" action="{% url 'staff:evaluation_textanswers_update_publish' %}" id="textanswer-update-form">
+        {% csrf_token %}
+
+        <input type="hidden" name="evaluation_id" value="{{ evaluation.id }}" />
```

Comment:
With the `answer_id` inputs, there is only ever one that is not disabled, this one will then be sent with the form. Initially they are all disabled, because javascript will run the code that does the initial update of some of this state anyways (and enables the first while doing so).

The `evaluation_id` however is equal for all textanswers, so it is never disabled